www/app/docs/getting-started/[[...slug]]/languages/nextjs.mdx

```diff
@@ -218,7 +216,8 @@ Instead of using SQL `where` clauses to get only todos for this specific tenant,
 
 ```typescript
 configureNile(cookies().get("authData"), params.tenantid);
-const todos = await nile.db("todos").select("*"); // no need for where clause because we previously set Nile context
+-- no need for where clause because we previously set Nile context
```

Comment:
This is incorrect. Typescript comments are `//`

Reply:
fixed